src/InterventionBackend.php

```diff
@@ -812,9 +820,11 @@ public function __destruct()
      */
     public static function flush()
     {
-        /** @var CacheInterface $cache */
-        $cache = Injector::inst()->get(CacheInterface::class . '.InterventionBackend_Manipulations');
-        $cache->clear();
+        if (static::$flush_enabled) {
```

Comment:
I think we may want to fetch this from configs explicitly (e.g. `Config::get`)

Reply:
Completed with $this->config() to be consistent with the rest of the code in this file.

Do we want any @deprecated notices adding for v5?